Solutions/Flare/Analytic Rules/FlareInfectedDevice.yaml

```diff
@@ -0,0 +1,23 @@
+id: 9cb7c337-f170-4af6-b0e8-b6b7552d762d
+name: Flare Infected Device
+description: |
+    'Infected Device found on darkweb or Telegram'
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: Flare
+    dataTypes:
+      - Firework_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1555
+query: |
+  Firework_CL
+  | where category_name_s contains "Infected Device" or source_s "genesis_market" and (risk_score_d == "3" or risk_score_d == "4" or risk_score_d == "5")
```

Comment:
please correct this query 
It should be [ source_s contains "geneis_market" ]   **OR**  [ source_s == "geneis_market" ]